front/app/containers/Admin/projects/project/general/components/GeographicAreaInputs.tsx

```diff
@@ -1,78 +1,148 @@
-import React from 'react';
-
-// components
+import React, { useState } from 'react';
```

Comment:
Pulled this out of the form, because there were issues with useEffect related to areas. See comments lower down the pages for more details.